build.cake

```diff
@@ -16,7 +16,6 @@ Task("Clean")
     .Does(() =>
     {
         CleanDirectory("nuget");
-        CleanDirectory("packages");
```

Comment:
why?

Reply:
there are no folder packages who is create